app/observers/comment_observer.rb

```diff
@@ -1,5 +1,5 @@
 class CommentObserver < ApplicationObserver
-  def after_save(comment)
+  def after_create(comment)
```

Comment:
I've changed this to `after_create` because whenever a comment is created, and reaction for it is generated, and it will resave the comment, resulting in two slack notifications.